File: Entornos de Desarrollo/calculadora/Calculadora/MetodosCalculadora.java

public class MetodosCalculadora {
	double numero1,numero2,resultado;
	
	public MetodosCalculadora(double n1, double n2){
		numero1 = n1;
		numero2 = n2;
		resultado = 0;
	}
	
	public double getResultado(){
		return resultado;
	}
	
	public double getSuma(){
		resultado = numero1 + numero2;
		return resultado;
	}
	
	public double getResta(){
		resultado = numero1 - numero2;
		return resultado;
	}
	
	public double getMultiplicacion(){
		resultado = numero1 * numero2;
		return resultado;
	}
	
	public double getDivision(){
		resultado = numero1 / numero2;
		return resultado;
	}

}

File: Entornos de Desarrollo/c#/bank/Calculadora/Calculadora/MetodosCalculadora.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Calculadora
{
    public class MetodosCalculadora
    {

        double numero1;

        double numero2;

        double resultado;

        public MetodosCalculadora(double n1, double n2)
        {
            this.numero1 = n1;
            this.numero2 = n2;
            this.resultado = 0;
        }

        public double getResultado()
        {
            return this.resultado;
        }

        public double getSuma()
        {
            this.resultado = (this.numero1 + this.numero2);
            return this.resultado;
        }

        public double getResta()
        {
            this.resultado = (this.numero1 - this.numero2);
            return this.resultado;
        }

        public double getMultiplicacion()
        {
            this.resultado = (this.numero1 * this.numero2);
            return this.resultado;
        }

        public double getDivision()
        {
            this.resultado = (this.numero1 / this.numero2);
            return this.resultado;
        }
    }
}
